resolvers/manatee-primary-resolver/src/bin/echo-resolver.rs

```diff
@@ -0,0 +1,130 @@
+//
```

Comment:
I have a couple of suggestions for a perhaps better place to put this. We could make it an example program which I think would work since it interacts with the manatee-primary-resolver via its standard public API. Another option is to make it a separate crate (perhaps under `tools` at the top of the workspace?). With either option we get to keep the manatee-primary-resolver as a simple library crate and we can avoid having to bring in some of the extra build dependencies that are only need by the echo-resolver program. What do you think?

Reply:
This is a good idea! I went with the latter suggestion and gave it its own crate under a new `tools` directory at the top of the workspace.